src/Tribe/Commerce/PayPal/Main.php

```diff
@@ -2966,21 +3020,21 @@ public function get_attendee( $attendee, $post_id = 0 ) {
 			}
 		}
 
-		$attendee_data = array_merge( $this->get_order_data( $attendee->ID ), array(
-			'optout'       => $optout,
-			'ticket'       => $product_title,
-			'attendee_id'  => $attendee->ID,
-			'security'     => $security,
-			'product_id'   => $product_id,
-			'check_in'     => $checkin,
-			'order_status' => $status,
-			'user_id'      => $user_id,
-			'ticket_sent'  => $ticket_sent,
+		$attendee_data = array_merge( $this->get_order_data( $attendee->ID ), [
```

Comment:
* :no_entry_sign: Opening parenthesis of a multi-line function call must be the last content on the line
* :no_entry_sign: Only one argument is allowed per line in a multi-line function call

<sub>hash: e4401f7abe762beb3ef83827c632de34</sub>